alfalfa_web/server/api.js

```diff
@@ -78,21 +62,24 @@ class AlfalfaAPI {
         const model = await this.models.findOne({ _id: run.model });
         let site = await this.sites.findOne({ ref_id: siteRef });
 
+        const site_dict = {
+          id: siteRef,
+          name: model.model_name,
+          status: run.status.toLowerCase(),
+          datetime: "",
+          simType: run.sim_type,
+          uploadTimestamp: run.created,
+          uploadPath: `uploads/${model.ref_id}/${model.model_name}`,
+          errorLog: run.error_log
+        };
+
         if (site) {
-          site = mapHaystack(site);
```

Comment:
We probably want to keep the haystack mapper, right now the names have `s:` prefixes unless there's a preprocessing error: 
![image](https://user-images.githubusercontent.com/411466/231580491-06edccb7-d5d6-410e-80c1-b060e409ced1.png)

Reply:
ahh, so that's why the haystack call was there.